stl/inc/utility

```diff
@@ -56,7 +56,7 @@ void swap(_Ty (&_Left)[_Size], _Ty (&_Right)[_Size]) noexcept(_Is_nothrow_swappa
     }
 }
 
-template <class _Ty, class>
+template <class _Ty, int>

```

Comment:
To avoid declaration/definition mismatch, this will need to be ...

```suggestion
#if _HAS_CXX17
template <class _Ty, enable_if_t<is_move_constructible_v<_Ty> && is_move_assignable_v<_Ty>, int>>
#else // ^^^ _HAS_CXX17 / !_HAS_CXX17 vvv
template <class _Ty, int>
#endif // _HAS_CXX17
```

Reply:
Yeah that one is needed, although I have no idea why